packages/visual-editor/src/components/Dropzone/Dropzone.types.ts

```diff
@@ -0,0 +1,6 @@
+import { CompositionComponentNode } from '@contentful/experience-builder-core/types';
+
+export type RenderDropzoneFunction = (
```

Comment:
If the `renderDropZone` was in its own hook, this could be added to that hook to keep the type and the definition together.